packages/flutter_test/lib/src/controller.dart

```diff
@@ -143,7 +143,7 @@ class WidgetController {
   ///
   /// * Use [firstState] if you expect to match several states but only want the first.
   /// * Use [stateList] if you expect to match several states and want all of them.
-  T state<T extends State>(Finder finder) {
+  T state<T extends State>(Finder finder) { // ignore: strong_mode_not_instantiated_bound
```

Comment:
Ok, that's good to know.  In general that's what we do (that is, fill in with the bound), but we're being a bit more conservative with types used in bounds.

Would you expect to be able to use my example `Foo` above with no type arguments?  And if so what would you expect it to mean?